mgmt_cli_test.py

```diff
@@ -552,16 +553,30 @@ def _get_all_snapshot_files_gce(cluster_id, bucket_name):
         # Unlike S3, if no files match the prefix, no error will occur
         return file_set
 
+    @staticmethod
+    def _get_all_snapshot_files_azure(cluster_id, bucket_name):
+        file_set = set()
+        credentials = KeyStore().get_backup_azure_blob_credentials()
+        blob_service_client = BlobServiceClient(account_url=f"https://{credentials['account']}.blob.core.windows.net",
+                                                credential=credentials['key'])
+        container_client = blob_service_client.get_container_client(bucket_name)
+        blob_iterator = container_client.list_blobs(name_starts_with=f"backup/sst/cluster/{cluster_id}")
+        for listing_object in blob_iterator:
+            file_set.add(listing_object["name"])
+        return file_set
```

Comment:
Have you thought about using libcloud for the same task?

```python
credentials = KeyStore().get_backup_azure_blob_credentials()
azure_driver = libcloud.storage.providers.get_driver(libcloud.storage.types.Provider.AZURE_BLOBS)
driver = azure_driver(key=credentials["account"], secret=credentials["key"])
```
and rest is the same as for GCE.

Reply:
You're right, it's working finely!
Updating